apps/voting/app/src/components/VoteActions.js

```diff
@@ -0,0 +1,257 @@
+import React, { useState, useCallback } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  GU,
+  IconCheck,
+  IconCross,
+  Info,
+  RADIUS,
+  ExternalLink,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { useAppState, useConnectedAccount } from '@aragon/api-react'
+import { VOTE_NAY, VOTE_YEA, VOTE_ABSENT } from '../vote-types'
+import { useExtendedVoteData } from '../vote-hooks'
+import { isVoteAction, getVoteStatus } from '../vote-utils'
+import { noop, formatDate } from '../utils'
+
+const VoteActions = React.memo(({ vote, onVoteYes, onVoteNo, onExecute }) => {
+  const theme = useTheme()
+  const connectedAccount = useConnectedAccount()
+  const { tokenSymbol } = useAppState()
+  const { connectedAccountVote, data } = vote
+  const { snapshotBlock, startDate: startTimestamp, open, executed } = data
+  const { canUserVote, canExecute, userBalance } = useExtendedVoteData(vote)
+  const [changeVote, setChangeVote] = useState(false)
+  const handleChangeVote = useCallback(() => setChangeVote(true), [])
+  const hasVoted = [VOTE_YEA, VOTE_NAY].includes(connectedAccountVote)
+
+  if (!open) {
+    return (
+      <React.Fragment>
+        {canExecute && !executed && isVoteAction(vote) && (
+          <React.Fragment>
+            <Button mode="strong" onClick={onExecute} wide>
+              Enact this vote
+            </Button>
+            <Info>
+              The voting perdiod is closed and the vote status is passed.{' '}
+              <span css="font-weight: bold;">Anyone</span> can now enact this
```

Comment:
`em` makes things look like _italics_ no?

Reply:
I don't mind either, just want to make sure we're all aligned cc @dizzypaty

Reply:
Hmm yes, perhaps `strong` is better since `em` usually defaults to italics. [They're roughly the same semantically.](http://html5doctor.com/i-b-em-strong-element/)

Reply:
Please use strong to empathize.